content/dogepedia/articles/how-to-backup-a-wallet.md

```diff
@@ -15,9 +15,36 @@ Most Dogecoin holders use some kind of [*wallet application*](/dogepedia/article
 
 Dogecoin Core, the software used to run a Dogecoin node, is often used as a wallet application. In order to create a backup with Dogecoin Core, follow these steps:
 
-- (Optional, but recommended) First, encrypt your wallet. To do so, go to Settings -> Encrypt wallet. You will be asked to enter a password. It is crucial that you store this password somewhere safe and that you never lose it. Without it, you will be unable to access the funds in your encrypted wallet. On the other hand, this password adds an extra layer of security to your backup. Without it, even if an attacker gains access to your wallet file, he will still be unable to spend your Dogecoin.
+- (Optional, but recommended) First, encrypt your wallet. To do so, go to Settings -> Encrypt wallet. You will be asked to enter a password. It is crucial that you store this password somewhere safe and that you never lose it. Without it, you would be unable to access the funds in your encrypted wallet. 
+
+   On the other hand, this password adds an extra layer of security to your backup. Without it, even if an attacker gains access to your wallet file, he will still be unable to spend your Dogecoin, unless he succeeds in brute forcing or guessing the encryption password.

```

Comment:
Line 20: Consider revising all instances of the term "he" to gender neutral language (e.g., "they"). If changed to "they", revise the word "succeeds" to the word " succeed".